basics/indexing/forward-index.md

```diff
@@ -1,15 +1,29 @@
 # Forward Index
 
-The values for every column are stored in a forward index, of which there are three types:
+At conceptual level, the forward index is a map from document id (aka row index) to the actual column value of the row.
+How this map is implemented depends on the index encoding and on whether the column is sorted or not.
 
-* [Dictionary encoded forward index](forward-index.md#dictionary-encoded-forward-index-with-bit-compression-default)\
-  Builds a dictionary mapping 0 indexed ids to each unique value in a column and a forward index that contains the bit-compressed ids.
-* [Sorted forward index](forward-index.md#sorted-forward-index-with-run-length-encoding)\
-  Builds a dictionary mapping from each unique value to a pair of start and end document id and a forward index on top of the dictionary encoding.
-* [Raw value forward index](forward-index.md#raw-value-forward-index)\
-  Builds a forward index of the column's values.
+When encoding is `RAW`, the forward index is implemented as an array whose indexes are the document ids and the values
+are the actual row value. 
+Read the [raw value forward type](forward-index.md#raw-value-forward-index) to learn more about that.
 
-To save segment storage space the forward index can now be [disabled](forward-index.md#disabling-the-forward-index) while creating new tables.
+When the encoding is `DICTIONARY`, the forward index does not physically store the actual row values but the dictionary ids.
+This means that there is an extra indirection each time a value has to be read, but enable two more efficient physical 
+layouts depending on whether the segment is sorted by this column or not.
+The [dictionary encoded forward index](forward-index.md#dictionary-encoded-forward-index-with-bit-compression-default) 
+and [sorted forward index](forward-index.md#sorted-forward-index-with-run-length-encoding) explain each option in detail.
+
+## Configuration
+
+Forward indexes are enabled by default.
+This means that columns will contain a forward index unless they are explicitly disabled.
+By doing so, most scanning access to the data will be disabled, but this is useful when other indexes are sufficient to cover the required data patterns.
```

Comment:
When the forward index is enabled, most scanning...?

Reply:
Here `By doing so` refers to the act of disabling the index. If we remove that using the previous suggestion we would need to rephrase this line.